performance-comparisons/v9.8/results/README.md

```diff
@@ -0,0 +1,21 @@
+# Mattermost load-test-ng
```

Comment:
I think this whole file got added inadvertently! We should remove it from this PR.